src/Traits/HasFactor.php

```diff
@@ -3,11 +3,17 @@
     
 trait HasFactor
 {
+    /**
+     * @return double
```

Comment:
Yeah, php can be funny sometimes. It originates from the [gettype](https://secure.php.net/manual/en/function.gettype.php) method which returns double instead of float